src/services/preProcess.ts

```diff
@@ -293,10 +293,18 @@ namespace ts {
             //    export import i = require("mod")
             //    (for JavaScript files) require("mod")
 
+            // Do not look for:
+            //    AnySymbol.import("mod")
+
             while (true) {
-                if (scanner.getToken() === SyntaxKind.EndOfFileToken) {
+                const token = scanner.getToken();
+                if (token === SyntaxKind.EndOfFileToken) {
                     break;
                 }
+                if (token === SyntaxKind.DotToken) {

```

Comment:
what about `a.b.import`?

Reply:
Should still work, because the next token is a literal, which means that it will go into the else came - I will add a test!

Reply:
4dc9db1e70